src/verify.py

```diff
@@ -0,0 +1,14 @@
+N=[3,4,5,1,2,3,4,9,13,0]
+
+def verify(a):
+	for item in N:
+		if int(a) == int(item):
+			return True
+	return False
+
+print("Inserire elemento da cercare:")
+x=input()
+if verify(x):
+	print("L'elemento è nell'array")
+else:
+	print("L'elemento non è nell'array")
```

Comment:
È buona pratica evitare di lasciare parti dello script senza indentazione,
perché verrebbero eseguiti ogni volta che qualcuno importa il tuo script.
Per evitarlo, includi tutte queste righe in una funzione 'main' e aggiungi un 
```python
if __name__ == "__main__":
    main()
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do).